packages/swapkit/chainflip/package.json

```
@@ -49,4 +49,4 @@
   "type": "module",
   "types": "./src/index.ts",
   "version": "1.0.0-rc.68"
-}
+}
```

Seems as your env is removing spaces (not integrated with proj). Check it out in free time
```suggestion
}

```